src/js/records/operations/issuance.record.js

```diff
@@ -12,7 +12,15 @@ export class IssuanceRecord extends OpRecord {
     this.percentFee = record.feePercent
     this.subject = record.reference
     this.date = record.ledgerCloseTime
-    this.accountId = accountId
+
+    // TODO: fix (merge conflict quick fix)
```

Comment:
this is my shit. gonna include it into "Refactor records" task

Reply:
ok